addons/main/XEH_postInit.sqf

```diff
@@ -75,7 +75,7 @@ if (GVAR(aceMedicalLoaded)) then {
     ["CAManBase", "HandleHeal", {
         [{
             (_this select 0) setDamage 0;
-            _this call FUNC(handleHealEh);
+            (_this + [true]) call FUNC(handleHealEh);
```

Comment:
do params instead and pass each value on its own in a new array. the `+[true]` is a little confusing